module2/advanced-css.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+		<title>Advanced CSS Pt 1</title>
+
+		<link rel="stylesheet" href="reveal.js/dist/reset.css">
```

Comment:
The Reveal framework files and CSS are not working properly here and just need an update on the file paths. Since this is inside of the modules folder the JavaScript and CSS references need to be reformatted with ```../<module>/<filename.html>```.

Line 9-15, 26-29 all need updated relative links.